inspirehep/modules/records/serializers/__init__.py

```diff
@@ -98,6 +99,20 @@
     'application/json',
 )
 
+json_authors_ui_v1 = JSONSerializer(
+    AuthorsRecordSchemaJSONUIV1
+)
+
+json_authors_ui_v1_search = search_responsify(
+    json_authors_ui_v1,
+    'application/vnd+inspire.literature.ui+json'
+)
+
+json_authors_ui_v1_response = record_responsify_nocache(
+    json_authors_ui_v1,
+    'application/vnd+inspire.literature.ui+json'
```

Comment:
this are for the authors, I'd change the header to ``'application/vnd+inspire.authors.ui+json'`` cc @harunurhan